lib/Ohmbrewer_RIMS.h

```diff
@@ -105,6 +85,24 @@ namespace Ohmbrewer {
             // friend std::ostream& operator<<( std::ostream& os, RIMS const& rims);
 
             /**
+             * The Tube thermostat
+             * @returns The Thermostat object representing the RIMS tube elements
+             */
+            Thermostat* getTube() const;
+
+            /**
+             * The temperature sensor located in the tun
+             * @returns The Temperature Sensor object representing the sensor located in the mash tun
+             */
+            TemperatureSensor* getTunSensor() const;
```

Comment:
Isn't this getSafetySensor now?

Remember we've decided that the sensor within the Tube object shall actually be placed in the thing it's trying to control the temperature of (the mash tun) while the second sensor (the one that's actually inline with the tube) shall be the Safety Sensor.

I know this is a little confusing since we've named the Thermostat object within the RIMS "Tube" - if you can think of a better name ("Controller" maybe?), I'm down.